tests/tck/features/bugfix/SubgraphBeforePipe.feature

```diff
@@ -11,7 +11,7 @@ Feature: Test get subgraph before pipe
   Scenario: subgraph with limit
     When executing query:
       """
-      GET SUBGRAPH WITH PROP FROM 'Tim Duncan' | LIMIT 1
+      GET SUBGRAPH WITH PROP FROM 'Tim Duncan' YIELD VERTICES as nodes, EDGES as relationships | LIMIT 1
```

Comment:
Maybe we could try `vertices_`, `edges_` to avoid the keyword confliction.

Reply:
don't agree , we should compatible with the previous version

Reply:
But it's incompatible now, previous version support subgraph without yield, now force it...

Reply:
also support subgraph without yield now

Reply:
Do you test it? I see the all test query has yield clause.

Reply:
in subgraph.feature